charts/shoot-addons/charts/kubernetes-dashboard/templates/deployment-metrics-scraper.yaml

```diff
@@ -0,0 +1,54 @@
+{{- if semverCompare ">= 1.16" .Capabilities.KubeVersion.GitVersion }}
+---
+apiVersion: {{ include "deploymentversion" . }}
+kind: Deployment
+metadata:
+  name: dashboard-metrics-scraper
+  namespace: kubernetes-dashboard
+  labels:
+    garden.sapcloud.io/role: optional-addon
+    origin: gardener
+    k8s-app: dashboard-metrics-scraper
+spec:
+  revisionHistoryLimit: 0
+  replicas: 1
+  selector:
+    matchLabels:
+      k8s-app: dashboard-metrics-scraper
+  template:
+    metadata:
+      labels:
+        garden.sapcloud.io/role: optional-addon
+        k8s-app: dashboard-metrics-scraper
+        origin: gardener
+      annotations:
+        seccomp.security.alpha.kubernetes.io/pod: 'runtime/default'
+    spec:
+      containers:
+      - name: dashboard-metrics-scraper
+        image: {{ index .Values.images "kubernetes-dashboard-metrics-scraper" }}
+        ports:
+          - containerPort: 8000
+            protocol: TCP
+        livenessProbe:
+          httpGet:
+            scheme: HTTP
+            path: /
+            port: 8000
+          initialDelaySeconds: 30
+          timeoutSeconds: 30
+        volumeMounts:
+        - mountPath: /tmp
+          name: tmp-volume
+        securityContext:
+          allowPrivilegeEscalation: false
+          readOnlyRootFilesystem: true
+          runAsUser: 1001
+          runAsGroup: 2001
+      serviceAccountName: kubernetes-dashboard
+      nodeSelector:
+        "beta.kubernetes.io/os": linux
```

Comment:
This label is deprecated since `1.14` can you create function that returns `kubernetes.io/os` for  versions >= `1.14`?

Reply:
same as above